IBAnimatable/MaskDesignable.swift

```diff
@@ -50,7 +51,9 @@ public extension MaskDesignable where Self: UIView {
     case .triangle:
       maskTriangle()
     case .none:
-      layer.mask?.removeFromSuperlayer()
+      if previousMaskType != nil {
+        layer.mask?.removeFromSuperlayer()
```

Comment:
Just wondering, don't you need to set `previousMaskType` to nil once the layer is removed?